flowmachine/flowmachine/features/utilities/unique_values_from_queries.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Given a list of queries and a list of columns, returns a table of each unique combination of `column` .
+"""
+
+from typing import List
+
+from flowmachine.core.query import Query
+from flowmachine.core.errors import MissingColumnsError
+
+
+class UniqueValuesFromQueries(Query):
+    """
+    Class representing unique values in  across a set of queries
+
+    Parameters
+    ----------
+    query_list: List[Query]
+        A list of Query objects, each of which must include a `subscriber` column
+
+    """
+
+    def __init__(self, query_list: List[Query], column_names: List[str]):
```

Comment:
```suggestion
    def __init__(self, query_list: List[Query], *, column_names: List[str]):
```
Good to make the 'column_names' arg keyword-only, I think

Reply:
Probably not a bad idea to do both

Reply:
Yes, no harm in making them both keyword-only. I think you're right that it doesn't make sense to have a default for `column_names`.